app/controllers/application_controller.rb

```diff
@@ -20,6 +20,8 @@ def conversation
   protected
 
   def configure_permitted_parameters
+    params = [:name, :email, :password, :password_confirmation]
```

Comment:
No need for all of the params, as Devise has built in support for everything except `:name`

Reply:
done